### PR TITLE
Add code to switch scenes

### PR DESCRIPTION
**Pull request by nonameentername**

### CODE REVIEW

File: player/player.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+using Godot;
+using System;
+
+public class player : KinematicBody2D
```
Comment (dmendiza) [inline player/player.cs]: Is this file needed?  I think only `Player.cs` is used.

Comment (nonameentername) [reply player/player.cs]: I removed the file
